src/st/no2_psp/e_init.c

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "../no2/no2.h"
+
+void func_pspeu_09256D08(Entity* self);
```

Comment:
If you map this name to the us name as I did with DAI it makes segment matching really easy, as well as allowing immediate merging of e_init.c